CMakeLists.txt

```diff
@@ -282,7 +282,7 @@ endif()
 if(SFML_BUILD_TEST_SUITE)
     if (SFML_OS_IOS)
         message( WARNING "Unit testing not supported on iOS")
-    else()
+    elseif(SFML_BUILD_WINDOW AND SFML_BUILD_GRAPHICS AND SFML_BUILD_NETWORK AND SFML_BUILD_AUDIO)
```

Comment:
I think we should output at least a warning message, that tests aren't being executed.
Otherwise someone might enable tests and just build some modules and not realize tests aren't even being executed.

Reply:
Good idea. I just added a warning for that.